lib/modules/dosomething/dosomething_image/dosomething_image.install

```diff
@@ -18,3 +18,10 @@ function dosomething_image_schema() {
 function dosomething_image_update_7001(&$sandbox) {
   field_delete_field('field_video_poster');
 }
+
+/**
+ * Deletes field_video_id.
+ */
+function dosomething_image_update_7002(&$sandbox) {
+  field_delete_field('field_video_id');
```

Comment:
This is going to delete the field entirely from the DB, you'll see its used in other spots. You need to just delete the instance per #https://github.com/DoSomething/dosomething/issues/3877#issuecomment-72870165